src/components/AccountSettings.tsx

```diff
@@ -0,0 +1,190 @@
+import React, { useState, useEffect } from 'react'
+import { Button, Col, Container, FormControl, FormGroup, FormLabel, Row } from 'react-bootstrap'
+import { useSynapseContext } from 'synapse-react-client/dist/utils/SynapseContext'
+import { UserProfile, UserBundle } from 'synapse-react-client/dist/utils/synapseTypes'
+import { SynapseConstants } from 'synapse-react-client'
+import { getMyUserBundle, updateMyUserProfile } from 'synapse-react-client/dist/utils/SynapseClient'
+import { displayToast } from 'synapse-react-client/dist/containers/ToastMessage'
+import StarterAccount from '../assets/StarterAccount.svg'
+import VerifedAccount from '../assets/VerifiedAccount.svg'
+import EditIcon from '../assets/RedEditPencil.svg'
+import ChangePasswordPage from './ChangePassword'
+import { ORCiDButton } from './ORCiDButton'
+export type AccountSettingsProps = {
+}
+
+const AccountSettings = (props: AccountSettingsProps) => {
+    const { accessToken } = useSynapseContext()
+    const [ userProfile, setUserProfile ] = useState<UserProfile>()
+    const [ orcid, setOrcid ] = useState<string>()
+    const [ verified, setVerfied ] = useState<boolean>()
+    const [ editUsername, setEditUsername] = useState<boolean>()
```

Comment:
You could initialize these edit fields to false to avoid the !!! in the code below.